examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -0,0 +1,232 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class FortnightlyDemo extends StatelessWidget {
+  static const String routeName = '/fortnightly';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Typography Demo',
```

Comment:
Is this really the title you want?

Reply:
Made it `Fortnightly Demo`. If there's no AppBar, where else does the title show up?

Reply:
It's the app's title when the OS suspends it.

Reply:
Even when it's a child of another `MaterialApp`? Does it use the lowest child that's visible?

Reply:
All of the gallery apps were written so that one could easily lift the app out (of the gallery) and run standalone. That was the original idea anyway. So specifying a sensible title is useful.

Reply:
Ah! Word.